build.gradle.kts

```diff
@@ -31,6 +32,7 @@ plugins {
 repositories {
     mavenCentral()
     jcenter()
+    maven(url = "http://packages.confluent.io/maven/")
```

Comment:
Denne trenger vi for å kunne hente Kafka-Schema-Registry, som Kafka-Embedded-Env er avhengig av.